policyengine_us/variables/gov/ssa/ssi/eligibility/income/deemed/from_ineligible_spouse/ssi_income_deemed_from_ineligible_spouse.py

```diff
@@ -22,8 +22,6 @@ def formula(person, period, parameters):
             "ssi_unearned_income_deemed_from_ineligible_spouse", period
         )
 
-        amount = parameters(period).gov.ssa.ssi.amount
```

Comment:
deleted on purpose?

Reply:
yes the the same parameter is defined below as `ssi`